bundle/Controller/Content/UpdateAlwaysAvailable.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Controller\Content;
+
+use Ibexa\Contracts\AdminUi\Controller\Controller;
+use Ibexa\Contracts\Core\Repository\ContentService;
+use Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException;
+use Ibexa\Core\Helper\TranslationHelper;
+use LogicException;
+use Netgen\Bundle\BetterIbexaAdminUIBundle\Form\FormFactory;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Contracts\Translation\TranslatorInterface;
+
+final class UpdateAlwaysAvailable extends Controller
+{
+    public function __construct(
+        private readonly TranslatorInterface $translator,
+        private readonly ContentService $contentService,
+        private readonly FormFactory $formFactory,
+        private readonly TranslationHelper $translationHelper
+    ) {
+    }
+
+    public function __invoke(Request $request): Response
+    {
+        $form = $this->formFactory->createContentAlwaysAvailableUpdateForm();
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            /** @var \Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Data\Content\ContentUpdateAlwaysAvailableData $data */
+            $data = $form->getData();
+
+            $contentInfo = $data->getContentInfo();
+            $alwaysAvailable = $data->getAlwaysAvailable();
+
+            if ($contentInfo === null || $alwaysAvailable === null) {
+                throw new LogicException(
```

Comment:
BadRequestHttpException?

Reply:
Updated in https://github.com/netgen/better-ibexa-admin-ui/pull/4/commits/901676109914e197dace0855011cd12c646cb929.